test/integration/framework/etcd.go

```diff
@@ -74,7 +74,8 @@ func startEtcd(output io.Writer) (func(), error) {
 	}
 	klog.V(1).Infof("could not connect to etcd: %v", err)
 
-	currentURL, stop, err := RunCustomEtcd("integration_test_etcd_data", nil, output)
+	etcdArgs := []string{"--experimental-watch-progress-notify-interval", "5s"}
```

Comment:
@jpbetz @serathius any concern about how this experimental flag impacts the results of the integration tests

Reply:
I've been tracking this. No concerns for me.

Reply:
+1 - this is fine, we should have done that long time ago TBH